### ISSUE TITLE
iExecutor not working

### ISSUE THREAD

**Issue by ImTheStigHonest** [scalda/FoolCraft#536]
## SUGGESTION/HELP
- [ ] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

whenever you exit the guy the settings reset to default
selecting right click doesn't accomplish anything
world crashes if enderio item conduits try to input items

open gui select right click select input/output
close gui no right click function happening
open gui settings back at default

block has power and has been tried with and without red stone signal

[LGH9eTdP.txt](https://github.com/scalda/FoolCraft/files/1066349/LGH9eTdP.txt)

**Comment by datitisev** [comment]
This is a duplicate of #522.

Also, I looked at the mod's changes, and it looks like it's been fixed in the latest release (https://github.com/Zairus/iExtras/commit/2ff7fac0a26011339187df9bd167a316c35d613f).

The mod will probably be updated for Foolcraft v1.4.1

**Comment by datitisev** [comment]
Wow, seems like Foolcraft v1.4.1 was just released 4 hours ago.

**Comment by ImTheStigHonest** [comment]
thats why I raised it again as this is running 1.4.1

**Comment by datitisev** [comment]
Ah, ok, Sorry about that. Still, you should probably continue the discussion over at #522.

**Comment by ImTheStigHonest** [comment]
dont place double crop sticks at the moment it still is an issue :0

**Comment by datitisev** [comment]
I can confirm this still doesn't work in 1.4.3

**Comment by ImTheStigHonest** [comment]
crop sticks work though :)

**Comment by Jupisoft111** [comment]
I just tried it in Foolcraft 1.4.4 and the interface still loose it's settings upon closing it, so hopefully soon will be fixed. This mod really looks very promising :-)

**Comment by ImTheStigHonest** [comment]
confirmed 1.4.4 still losing settings

**Comment by scalda** [comment]
yes we are waiting on a fixed version from @WinterGrave

**Comment by ImTheStigHonest** [comment]
what issues should I avoid if I enable the mechanical user until we get the update?

**Comment by scalda** [comment]
What are you planning on using it for?

**Comment by ImTheStigHonest** [comment]
at the moment its for automating tree crops bonemeal on pam's harvest craft items such as olive's

**Comment by scalda** [comment]
enabling the user just for that should be ok
